### DISCUSSION TITLE
Matching Gene Trees in Treefile to Alignment File Names

### DISCUSSION THREAD

**Discussion by simone-says** [iqtree/iqtree2#323] [Q&A]
Is the order of the gene trees output in the *.treefile the same order that the alignment files are read in, or the same order that is output into the log file during partition testing? I need to match individual gene trees to the alignments that made them, but there are no unique identifiers preserved in the *.treefile. 

This is the command used to make the gene trees:
`iqtree2 -S /projects/busco_phylo/results/03_trimal/03_realigned_trim2/UCEs/05_unsat --prefix UCEs_unsat.loci -T AUTO`

**Comment by roblanf** [comment]
@simone-says, they are in the same order as the 

```
SEQUENCE ALIGNMENT
------------------
```

section of the `.iqtree` file. A simple way to confirm this if you have some trees with differening numbers of taxa is to cross-reference them that way (the table in the `SEQUENCE ALIGNMENT` section has taxon numbers).

Rob
